store/store.go

```diff
@@ -96,6 +106,17 @@ func (s *Store) Close() error {
 	return s.metrics.close()
 }
 
+func (s *Store) Cache(height uint64, acsr eds.AccessorStreamer) error {
```

Comment:
In order to get and cache accessor in blocks for you would need to open it twice.
1. Get accessor
2. Cache accessor
The first issue with this approach is that it is inefficient. The second, it is not concurrency safe, because parallel Get call may result in duplicate Put of same accessor in the cache. GetOrLoad method in cache allows to avoid both issues, if used directly by blockstore.

Reply:
Why is it inefficient?

Reply:
There is no concurrency issue because Cache has height based locking and if two Puts happen simultaneously the other one will resolve gracefully with Get portion instead of Load

Reply:
It is inefficient because for every get you need to open file 2 times. When you don’t need to if only getorload is used.
Concurrency issue happens when 2 parallel get happens. They will load different accessors  with independent internal axis caches. Then you will load third one into cache by Cache method. Third accessor will be cached, while read will happen from first and second one. No axis cache proofs from first and second will be kept in third(chached) accessor